### DISCUSSION TITLE
Example needed to migrate ForAllOtherMembers to use MemberList.None

### DISCUSSION THREAD

**Discussion by gopala000** [LuckyPennySoftware/AutoMapper#4242] [NA]
### Source/destination types

```csharp
// Put your source/destination types here
```

### Mapping configuration
.ForAllOtherMembers(opt => opt.Ignore())

```csharp
// Mapper.Initialize or just the CreateMap snippet 
```

### Version: x.y.z
 
Migrating from 10.1.1 to 12.0.1

### Expected behavior
I'm migrating from 10.1.1 to 12.0.1 and need to get an equivalent of ForAllOtherMembers(opt => opt.Ignore()). Using CreateMap<Source, Destination>(MemberList.None) as given in https://docs.automapper.org/en/latest/11.0-Upgrade-Guide.html doesn't produce the same behavior.  Appreciate if you can provide a pointer to a working example to migrate ForAllOtherMembers().
 

I expect the migration path for ForAllOtherMembers(opt => opt.Ignore()).

### Steps to reproduce

```csharp
// Your calls to Mapper.Map or ProjectTo here, with source/destination objects constructed
```

**Comment by jbogard** [comment]
There isn't one. This behavior is something we never should have included. You'll have to find some other way or implement it yourself.
